include/dds/core/Reference.hpp

```diff
@@ -0,0 +1,336 @@
+/*
+ * Copyright 2010, Object Management Group, Inc.
+ * Copyright 2010, PrismTech, Corp.
+ * Copyright 2010, Real-Time Innovations, Inc.
+ * Copyright 2019, Proyectos y Sistemas de Mantenimiento SL (eProsima).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OMG_DDS_CORE_REFERENCE_HPP_
+#define OMG_DDS_CORE_REFERENCE_HPP_
+
+#include <dds/core/types.hpp>
+#include <dds/core/refmacros.hpp>
+
```

Comment:
Needed for DECLARE_TYPE_TRAITS macro in line 93
```suggestion
#include <dds/core/ref_traits.hpp>

```

Reply:
Already fixed :D